### DISCUSSION TITLE
Replace RBAC for cluster level permissions

### DISCUSSION THREAD

**Discussion by pashafirdous1** [strimzi/strimzi-kafka-operator#5434] [Q&A]
Hi ,
we want to limit the [RBAC permissions](https://github.com/strimzi/strimzi-kafka-operator/tree/0.22.1/install/cluster-operator) installed with Kafka-operator/kafka-components as we have only namespaced deployments of kafka. 

we have several clusterrole/clusterrolebindings in the cluster-operator and we would like to replace them with roles/rolebindings.
we error out when we change the RBAC.

Could we get some advice on limiting the RBAC permissions to namespace level?

regards

**Comment by scholzj** [comment]
First of all, keep in mind that the cluster level permissions are there for a reason. So disabling them will mean that you will be unable to use some functionality ... such as rack-awareness (especially the rack-awareness is usually a must-have for production), node ports or volume resizing. So if you don't use the cluster level RBAC, you should not use these features either.

Normally, it should be easy - if you do not deploy the ClusterRole / ClusterRoleBinding resources, it should be able to detect it and not use them. So if you have some other error than not being able to use some features, please share more details.
